package.json

```diff
@@ -190,28 +182,28 @@
     "eslint-plugin-react": "^7.21.3",
     "eslint-plugin-react-hooks": "^4.1.2",
     "expose-gc": "^1.0.0",
-    "file-loader": "^6.1.0",
     "findup": "^0.1.5",
-    "fork-ts-checker-webpack-plugin": "^5.1.0",
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
     "get-port": "^5.1.1",
     "glob": "^7.1.6",
     "html": "^1.0.0",
     "html-format": "^1.0.1",
-    "html-webpack-plugin": "^4.4.1",
+    "html-webpack-plugin": "^5.5.1",
     "jest": "^24.1.0",
     "jest-cli": "^24.1.0",
     "moment": "^2.27.0",
     "moment-timezone": "^0.5.31",
+    "node-polyfill-webpack-plugin": "^2.0.1",
     "node-sass": "^8.0.0",
     "pegjs": "^0.10.0",
-    "postcss-cli": "^7.1.2",
-    "postcss-inline-svg": "^4.1.0",
-    "postcss-loader": "^4.0.1",
+    "postcss": "^8.4.23",
+    "postcss-cli": "^10.1.0",
+    "postcss-inline-svg": "^6.0.0",
+    "postcss-loader": "^7.3.0",
     "pre-commit": "^1.2.2",
     "prettier": "^2.1.2",
     "prop-types": "^15.6.0",
     "puppeteer": "^5.5.0",
-    "raw-loader": "^4.0.1",
```

Comment:
DEPRECATED for v5 (https://v4.webpack.js.org/loaders/file-loader/). It was replaced with `asset/source` type (https://webpack.js.org/guides/asset-modules/#replacing-inline-loader-syntax)